### PR TITLE
Implements listing and create

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

File: test/store/index.spec.js [hunk 1]
```diff
@@ -76,6 +83,43 @@ describe("store", () => {
...
+        }
+      }
+      const before = state.tasks.edges.length
+      store.mutations.PUSH_TASK(state, payload)
+      const after = state.tasks.edges.length
+      expect(after - before).toBe(1)
+    })
+
+    xit("MAKE_MUTATION_ID_TASK", () => {
+      store.mutations.MAKE_MUTATION_ID_TASK(state)
```
Comment (marocchino) [inline test/store/index.spec.js]: I don't know why but, uuid is not a function in this timing.

Comment (marocchino) [reply test/store/index.spec.js]: never mind -3-

File: src/components/TaskListItem.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  watch: {
+    instanceStatus: {
+      handler: 'onInstanceStatusChanged',
+      immediate: true,
+      deep: false
+    }
+  },
+  methods: {
+    onInstanceStatusChanged (newVal) {
+      this.$emit('updateTask', { id: this.taskId, status: newVal })
```
Comment (riseshia) [inline src/components/TaskListItem.vue:42]: Not implemeted yet?

File: src/store/index.js [hunk 3]
```diff
@@ -1,22 +1,38 @@
...
+      if (state.loading) { return }
       commit('START_LOADING')
       const payload = await api.Tasks(variables)
       commit('UPDATE_TASKS', payload)
       commit('FINISH_LOADING')
+    },
+
+    async CREATE_TASK ({ state, commit }) {
+      if (state.loading) { return }
+      commit('START_LOADING')
```
Comment (riseshia) [inline src/store/index.js:30]: I'm just curious, Is there any reason to split process to lots of commit?

Comment (marocchino) [reply src/store/index.js:30]: I can not naming mutation a (START_LOADING + MAKE_MUTATION_ID_TASK) and mutation b (PUSH_TASK + UPDATE_NEW_TASK + FINISH_LOADING).

File: src/components/TaskListItem.vue (hunk 2 above)
Comment (marocchino) [reply src/components/TaskListItem.vue:42]: Yes. I bring this code from typescript branch.
Do you want erase this?

Comment (riseshia) [reply src/components/TaskListItem.vue:42]: Nope. I hope to check. It's ok to leave it

**Review by riseshia — APPROVED** [review]

**Comment by marocchino** [comment]
Thanks!

**Pull request merged**